### PR TITLE
Fix/s3 storage external user

### PR DESCRIPTION
**Pull request by ianopolous**

### CODE REVIEW

File: vendor/priors/gwt.js [hunk 1]
```diff
@@ -25,37 +25,28 @@ function fragmentToProps(fragment) {
 function getProm(url) {
```
Comment (kevodwyer) [inline vendor/priors/gwt.js:25]: what about the other methods that use Promise?

File: vendor/priors/gwt.js [hunk 2]
```diff
@@ -75,7 +66,11 @@ function postProm(url, data) {
 		resolve(new Int8Array(req.response));
             }
             else {
-		reject(req.getResponseHeader("Trailer"));
+		try {
+		    reject(req.getResponseHeader("Trailer"));
+		} catch (e) {
+		    reject("Error");
```
Comment (kevodwyer) [inline vendor/priors/gwt.js:72]: how about adding the actual exception to the error message?

File: vendor/priors/gwt.js (hunk 1 above)
Comment (ianopolous) [reply vendor/priors/gwt.js:25]: There's nothing wrong with the promise, it's just unnecessary

File: vendor/priors/gwt.js (hunk 2 above)
Comment (ianopolous) [reply vendor/priors/gwt.js]: done

**Pull request merged**